### PR TITLE
[DOC] Add missing Monorepo configuration

### PR DESCRIPTION
**Pull request by serut**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Improve documentation
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | 
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

The babel-eslint-parser should provide in its README the simple configuration required on monorepo environment, to avoid to repeat the eslint/babel config on every package 

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/13253"></a>

### CODE REVIEW

File: eslint/babel-eslint-parser/README.md [hunk 1]
```diff
@@ -91,6 +91,20 @@ module.exports = {
 };
 ```
 
+**Monorepo configuration**
+
+This configuration is usefull for monorepo as it avoids to repeat the configuration of babel and eslint on every package.
```
Comment (fedeci) [inline eslint/babel-eslint-parser/README.md:96]:
```suggestion
This configuration is useful for monorepo as it avoids to repeat the configuration of Babel and ESLint on every package.
```
Just some typos :)

File: eslint/babel-eslint-parser/README.md [hunk 2]
```diff
@@ -91,6 +91,20 @@ module.exports = {
...
+```
+{
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "babelOptions": {
+      "rootMode": "upward"
+    }
+  }
+}
+```
```
Comment (fedeci) [inline eslint/babel-eslint-parser/README.md:107]: I would use a js config example as in other parts of the docs.

Comment (serut) [reply eslint/babel-eslint-parser/README.md]: Yeah you're right, I've tested and `.eslintrc.js` are working as well as `.eslintrc`

File: eslint/babel-eslint-parser/README.md [hunk 3]
```diff
@@ -91,6 +91,21 @@ module.exports = {
 };
 ```
 
+**Monorepo configuration**
+
+This configuration is useful for monorepo as it avoids to repeat the configuration of Babel and ESLint on every package.
```
Comment (nicolo-ribaudo) [inline eslint/babel-eslint-parser/README.md:96]:
```suggestion
If you are running ESLint in specific packages and not from the monorepo root, this configuration lets `@babel/eslint-parser` load the Babel configuration defined in the monorepo root.
```

(You don't need this option if the CWD is the monorepo root when running ESLint)

Comment (serut) [reply eslint/babel-eslint-parser/README.md:96]: I see. Maybe :
This configuration is useful for monorepo, when you are running ESLint on every package and not from the monorepo root folder, as it avoids to repeat the configuration of Babel and ESLint on every package.

**Review by nicolo-ribaudo — APPROVED** [review]

**Comment by JLHwung** [comment]
Thanks!

**Pull request merged**